src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/HttpClient/NoOp.cs

```diff
@@ -0,0 +1,55 @@
+
+// Copyright 2020 New Relic, Inc. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using NewRelic.Agent.Api;
+using NewRelic.Agent.Api.Experimental;
+using NewRelic.Agent.Extensions.Providers.Wrapper;
+using NewRelic.SystemExtensions;
+
+namespace NewRelic.Providers.Wrapper.HttpClient
+{
+    public class NoOp : IWrapper
```

Comment:
How about we only include this wrapper for .NET Core? So:

```csharp
#if NETSTANDARD2_0

<entire contents of this file.>

#endif
```

Reply:
Ideally, we'd spend some effort on introducing the ability to specify a version in the xml and have the profiler read the version... Then we wouldn't need this wrapper at all.